Library/Homebrew/formula_installer.rb

```diff
@@ -66,6 +66,7 @@ def initialize(formula)
     @poured_bottle = false
     @pour_failed = false
     @start_time = nil
+    @fetched = false
```

Comment:
You probably need another instance variable here:
```suggestion
    @fetched = false
    @built = false
```